src/widget/OktaSignIn.js

```diff
@@ -153,6 +160,10 @@ var OktaSignIn = (function () {
       redirectUri: options.redirectUri
     }, options.authParams);
 
+    if (!authParams.issuer) {
```

Comment:
curious why not authjs take care of this default setting?

Reply:
In AuthJS 3.0, "baseUrl" has been deprecated and issuer is now required. Rather than push that requirement on the SIW, this line is ensuring issuer is set and converting baseUrl to issuer (unless issuer was already set).